app/styles/tasks.css

```diff
@@ -36,49 +42,116 @@
 
 /* progress bar */
 
+.wrapper-div{
+  margin: 1em;
+  padding: 1rem;
+}
+
 .progress-bar {
-  background-color: #BFADA3;
-  height: 15px;
-  border-radius: 20px;
+  background-color: #fff;
+  height: 40px;
+  border-radius: 25px;
   overflow: hidden;
-  box-shadow: 2px 0 10px inset rgba(0,0,0,0.2);
   margin-top: 16px;
```

Comment:
is it better to shift to rem?

Reply:
rem makes sense for font-size, any specific reason to use it for margin?